plugin-dev/Source/Sentry/Private/SentryModule.cpp

```diff
@@ -7,7 +7,7 @@
 #include "UObject/Package.h"
 #include "UObject/UObjectGlobals.h"
 #include "Interfaces/IPluginManager.h"
-#include "Modules/ModuleManager.h"
+#include "modules/ModuleManager.h"
```

Comment:
Let's keep the included directives without changes (probably an auto replacement issue here)

Reply:
```suggestion
#include "Modules/ModuleManager.h"
```